### PR TITLE
add helm directory

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/5980.

### CODE REVIEW

File: helm/bridge-operator-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          name: bridge-operator-configmap
+          items:
+            - key: config.yaml
+              path: config.yaml
+      - name: certs
+        hostPath:
+          path: /etc/ssl/certs/ca-certificates.crt
+      - name: flannel-env-files
+        hostPath:
+          path: /run/flannel/networks/
```
Comment (xh3b4sd) [inline helm/bridge-operator-chart/templates/deployment.yaml]: Mounting the env files here.

File: helm/bridge-operator-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        image: quay.io/giantswarm/bridge-operator:[[ .SHA ]]
+        volumeMounts:
+        - name: bridge-operator-configmap
+          mountPath: /var/run/bridge-operator/configmap/
+        - name: bridge-operator-secret
+          mountPath: /var/run/bridge-operator/secret/
+        - name: certs
+          mountPath: /etc/ssl/certs/ca-certificate.crt
+        - name: flannel-env-files
+          mountPath: /run/flannel/networks/
```
Comment (xh3b4sd) [inline helm/bridge-operator-chart/templates/deployment.yaml]: And here within the container.

File: helm/bridge-operator-chart/templates/rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  - apiGroups:
+      - core.giantswarm.io
+    resources:
+      - flannelconfigs
+    verbs:
+      - "*"
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
```
Comment (xh3b4sd) [inline helm/bridge-operator-chart/templates/rbac.yaml]: Not sure about these.

Comment (MarcelMue) [reply helm/bridge-operator-chart/templates/rbac.yaml]: I don't see why these permissions are needed.

**Review by MarcelMue — APPROVED** [review]

File: helm/bridge-operator-chart/templates/rbac.yaml (hunk 3 above)
Comment (xh3b4sd) [reply helm/bridge-operator-chart/templates/rbac.yaml]: Be gone!

**Comment by xh3b4sd** [comment]
Helm magic works on `geckon`.

**Pull request merged**